client/my-sites/upgrades/domain-management/email/add-google-apps-card.jsx

```diff
@@ -29,14 +32,13 @@ const AddGoogleAppsCard = React.createClass( {
 	mixins: [ analyticsMixin( 'domainManagement', 'email' ) ],
 
 	render() {
-		const gapps = this.props.products.gapps,
+		const prices = get( this.props, 'products.gapps.prices', [] ),
```

Comment:
You could probably deconstruct `props` first, and then include the `currencyCode` here and default the price to 0 or something. If we somehow end w/o a price, might be some errors.